.github/workflows/pull-request-tasks.yml

```diff
@@ -0,0 +1,29 @@
+name: Swift Integration Tests
+
+on:
+  pull_request:
+    branches:
+  push:
+    branches:
+      - master
```

Comment:
let's also run on `develop`

Reply:
Are we enforcing GitFlow? The current setup is that all pull requests (no matter the base or target) will run tests, so if develop rejects direct pushes, we sort of have that covered. I specified pushes to master in case of hotfixes or patches that somehow skip develop (which would then mean we need to rebase develop on top of master again).

Will update none the less 👍🏾